### PR TITLE
Push code for review

### PR DESCRIPTION
**Pull request by Sahar-SE**

### CODE REVIEW

File: FCC1.html [hunk 1]
```diff
@@ -0,0 +1,106 @@
+<<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <title id="title">
+    Dr. Norman Borlaug
+  </title>
+<style type="text/css">
```
Comment (Sahar-SE) [inline FCC1.html:9]: It would be better if you write your CSS codes in separate file to increase performance.

File: FCC1.html [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+  text-align: center;
+}
+
+#em {
+  text-align: center;
+}
+
+  </style>
+  
+</head>
```
Comment (Sahar-SE) [inline FCC1.html:68]: Please use 2 space instead of tab to make your code cleaner and easier to read

File: FCC1.html [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+        <p id="sub">The man who saved a billion lives</p>
+       <div id="img-caption">
+         
+    
+  <img src="http://static1.squarespace.com/static/5256f6e4e4b086e2b62842e2/t/54bbbd61e4b077c90273b411/1421589862757/"id="image">
+        
+
+
+
+        <p id="tribute-info">
```
Comment (Sahar-SE) [inline FCC1.html:82]: Please put the Biography in an article tag to use the html semantics and the article tag is used to narrate something or add the Bio or code in our project.

File: FCC1.html (hunk 1 above)
Comment (topeogunleye) [reply FCC1.html:9]: Great job being polite, phrasing your comment as a suggestion, and giving a reason for the requested change

File: FCC1.html (hunk 2 above)
Comment (topeogunleye) [reply FCC1.html:68]: Great job being polite and giving a reason for the requested change

File: FCC1.html (hunk 3 above)
Comment (topeogunleye) [reply FCC1.html:82]: - [x] [ OPTIONAL] Great job mentioning this change but I kindly suggest that you phrase your comments as suggestions, be polite, and give a reason for the requested change

**Review by topeogunleye — CHANGES_REQUESTED** [review]
> # Changes Required 
> ### Hi @Sahar-SE,
> Nice job, you are almost there.
> 
> 
> 
> ## (Highlights) Good Points: 
- [x]  Review comments are polite, use suggestions and give reasons for requested change :heavy_check_mark:
>
> ## (Changes Required) Aspects to improve: 
> * [ ]  I kindly suggest that you make use of [Gihub inline comments feature](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/reviewing-proposed-changes-in-a-pull-request) to add a summary to your review
> See Demo video of above below:

https://user-images.githubusercontent.com/52934128/165086629-3307d613-4fc2-4659-bdb9-116aaa0000d9.mp4

> 
> 
> ## [Optional] suggestions:
> * Nothing to mention.
> 
> _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
> 
> **Cheers and Happy coding! :clap::clap::clap:**
> 
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear
> Please, remember to tag @topeogunleye in your question so I can receive the notification.
>
> **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
> 
> As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by Sahar-SE — COMMENTED** [review]
This is for my code reviewer, thank you for making sure I do an inline comment, and approve my project please.😊

**Review by vic778 — APPROVED** [review]
Hi @Sahar-SE , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Charlie Brown

## Highlights

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by topeogunleye** [comment]
> This is for my code reviewer, thank you for making sure I do an inline comment, and approve my project please.blush

Hi @Sahar-SE, you  need to add summary to your review. Here is How:

> This is for my code reviewer, thank you for making sure I do an inline comment, and approve my project please.blush

You're welcome. Kindly submit for a re-review please

**Pull request merged**